froide/foirequest/api_views/message.py

```diff
@@ -75,18 +78,6 @@ def publish(self, request, pk=None):
         if not message.is_response:
             message.sender_user = request.user
```

Comment:
Considering we have teams with write access this should not be set to the acting user. Maybe this field is actually not needed as in a request the only non-public body sender can ever officially be the requester? To record who actually acted on the request, we should use `FoiEvent` logging.